opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -111,6 +113,11 @@ def _init_logging(
             BatchLogProcessor(exporter_class(**exporter_args))
         )
 
+    log_emitter = provider.get_log_emitter(__name__)
+    handler = OTLPHandler(level=logging.NOTSET, log_emitter=log_emitter)
```

Comment:
Maybe a TODO to be able to configure logging levels?

Reply:
Probably not, by leaving it `NOTSET` we are already letting the users configure. Whatever level they set in application will be used for filtering.